Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -540,28 +540,37 @@ extension HTTPHeaders: RandomAccessCollection {
     }
 }
 
-extension Character {
+extension UTF8.CodeUnit {
     var isASCIIWhitespace: Bool {
-        return self == " " || self == "\t" || self == "\r" || self == "\n" || self == "\r\n"
+        switch self {
+        case UInt8(ascii: " "),
+             UInt8(ascii: "\t"),
+             UInt8(ascii: "\r"),
+             UInt8(ascii: "\n"):
+          return true
+
+        default:
+          return false
+        }
```

Comment:
Apparently two are missing:
- Vertical tab `\v`
- Form feed `\f`

Reply:
I'm disinclined to make this more complex. We're only really trying to catch the [`OWS` rule from the HTTP -semantics draft](https://www.ietf.org/archive/id/draft-ietf-httpbis-semantics-19.html#name-whitespace), which strictly forbids newline characters, and I don't think we really expect to see them, so arguably we could even remove the `\r` and `\n` cases. VTAB and form-feed should also be disregarded by this loop for the same reason.

Reply:
Sounds fine to me tbh, although it says `SHOULD` which I'm sure someone somewhere interprets as "not necessary"

Reply:
Sure, you can put a VTAB in there: the mistake would be for us to strip it, not for us to fail to. If your VTAB makes it to the app the app is naturally going to flip its lid: what we shouldn't do is treat it as though it was accidentally equivalent to SP or HTAB.

Reply:
Should we follow up with a PR to remove `\r` and `\n`?